packages/token-introspection/src/client/index.ts

```diff
@@ -49,7 +49,6 @@ export const createAxiosInstance = (args: {
     method: 'post',
     timeout: args.requestTimeoutMs
   })
-  axiosInstance.defaults.headers.common['Content-Type'] = 'application/json'
```

Comment:
removing because it is default already?

Reply:
I removed it because

1. it should've been `axiosInstance.defaults.headers.post['Content-Type'] = 'application/json'`
2. axios automatically uses `'application/json'` when the body is an object, which is now the case